iocage/lib/Config/Jail/File/Fstab.py

```diff
@@ -343,7 +349,8 @@ def add_line(
         ],
         skip_existing: bool=False,
         replace: bool=False,
-        auto_create_destination: bool=False
+        auto_create_destination: bool=False,
```

Comment:
I think so because you can either make sure yourself that the directory exists or you may set this neat flag. Consider this default to be interesting for library users but not so much in the CLI context.